lighty-modules/lighty-netconf-sb/src/main/java/io/lighty/modules/southbound/netconf/impl/NetconfTopologyPlugin.java

```diff
@@ -16,37 +16,57 @@
 import org.opendaylight.mdsal.dom.api.DOMMountPoint;
 import org.opendaylight.mdsal.dom.api.DOMMountPointService;
 import org.opendaylight.mdsal.dom.api.DOMRpcService;
+import org.opendaylight.mdsal.dom.api.DOMSchemaService;
 import org.opendaylight.netconf.client.NetconfClientDispatcher;
 import org.opendaylight.netconf.sal.connect.api.SchemaResourceManager;
 import org.opendaylight.netconf.sal.connect.impl.DefaultSchemaResourceManager;
 import org.opendaylight.netconf.sal.connect.netconf.schema.mapping.DefaultBaseNetconfSchemas;
 import org.opendaylight.netconf.topology.impl.NetconfTopologyImpl;
 import org.opendaylight.yang.gen.v1.urn.tbd.params.xml.ns.yang.network.topology.rev131021.NodeId;
 import org.opendaylight.yangtools.yang.data.api.YangInstanceIdentifier;
+import org.opendaylight.yangtools.yang.model.parser.api.YangParserException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 public class NetconfTopologyPlugin extends AbstractLightyModule implements NetconfSBPlugin {
 
-    private final NetconfTopologyImpl topology;
+    private static final Logger LOG = LoggerFactory.getLogger(NetconfTopologyPlugin.class);
+
     private final DOMMountPointService domMountPointService;
+    private final String topologyId;
+    private final NetconfClientDispatcher clientDispatcher;
+    private final AAAEncryptionService encryptionService;
+    private final LightyServices lightyServices;
+    private NetconfTopologyImpl topology;
```

Comment:
can be converted to local variable

Reply:
Nope, I don't think that will be a good idea to edit initProcedure() method. Do you see any other option how can be these convert to local variable?

Reply:
I mean, only `topology` variable, after refactoring constructor that should not be needed as a field

Reply:
done